### PR TITLE
Export component automation_email attribute

### PR DESCRIPTION
**Pull request by peikk0**

### CODE REVIEW

File: statuspage/resource_component.go [hunk 1]
```diff
@@ -124,7 +125,7 @@ func resourceComponent() *schema.Resource {
 			},
 			"only_show_if_degraded": &schema.Schema{
 				Type:        schema.TypeBool,
-				Description: "Requires a special feature flag to be enabled",
+				Description: "Should this component be shown component only if in degraded state",
```
Comment (yannh) [inline statuspage/resource_component.go]: How about "Only show this component if in degraded state" ?

**Comment by yannh** [comment]
LGTM, I might add an integration test later on (and see if I can run Gitlab's CI on PRs).
Thanks a lot for your contribution!

File: statuspage/resource_component.go (hunk 1 above)
Comment (peikk0) [reply statuspage/resource_component.go]: Sounds good.

Comment (peikk0) [reply statuspage/resource_component.go]: Sounds good.

**Pull request merged**